infra/terraform/modules/data_access/common/sentry/sentry/sentry.py

```diff
@@ -0,0 +1,26 @@
+'''
+Environment variables:
+ - ENV, e.g. "dev", "alpha", etc...
+ - SENTRY_DSN, Sentry DSN
+'''
+
+import os
+
+from raven import Client as Sentry
+from raven.transport.http import HTTPTransport as SentryHTTPTransport
+
+
+def catch_exceptions(fn):
```

Comment:
Minor naming quibble - to "catch" an exception suggests that it is handled, but this re-raises. Maybe `report_exceptions` or `log_exceptions` would be more self-explanatory.